### ISSUE TITLE
Large file curation and a11y review

### ISSUE THREAD

**Issue by ajkiessl** [psu-libraries/scholarsphere#1818]
For work versions that contain a PDF with more than 100 pages, two things should happen:

1.) A field in the record sent to AirTable should be set to `true` to indicate this work version has a large PDF
2.) The boolean indicating the work is under manual accessibility review (#1817) should be set to `true` on publish

**Comment by ajkiessl** [comment]
Is this so we don't break or overload our automated remediation process?  Like we don't want something too big going over to the remediation tool and taking up processing time, and space?  If so, then file size will probably work.  If it's more that admins want to manually do the remediation of PDFs with a lot of pages because we don't trust auto remediation, then I think we'll need to determine "largeness" from the page number.

**Comment by ajkiessl** [comment]
After doing a little digging, what causes a PDF to take up a lot of disk space is usually large, high-res, embedded images or it's just that the PDF is poorly generated.  Even PDFs that are hundreds of pages might not exceed a few MBs.  So, I'm thinking we'll need to take page count into consideration for this feature.

I updated the OP to reflect this.
